R/travis-basic.R

```diff
@@ -1,37 +1,39 @@
-#' Get repository information from Travis CI
+#' @title Get repository information from Travis CI
 #'
 #' @description
 #' Return repository information, in particular the repository ID.
 #'
-#' `travis_repo_info()` queries the "/repos/:repo" API.
+#' @import httr
+#'
+#' @details
+#' `travis_repo_info()` queries the `"/repos/:repo"` API.
 #'
 #' @param repo `[string|numeric]`\cr
 #'   The GitHub repo slug, by default obtained through [github_repo()].
 #'   Alternatively, the Travis CI repo ID, e.g. obtained through `travis_repo_id()`.
-#' @param token `[Token2.0]`\cr
-#'   A Travis CI token obtained from [travis_token()] or [auth_travis()].
-#'
 #' @seealso [Travis CI API documentation](https://docs.travis-ci.com/api)
 #'
 #' @family Travis CI functions
 #'
 #' @export
-travis_repo_info <- function(repo = github_repo(),
-                             token = travis_token(repo)) {
-  req <- TRAVIS_GET3(sprintf("/repo/%s", encode_slug(repo)), token = token)
-  httr::stop_for_status(req, sprintf("get repo info on %s from Travis", repo))
-  new_travis_repo(httr::content(req))
+travis_repo_info <- function(repo = github_repo()) {
+
+  req = travisHTTP(path = sprintf("/repo/%s", encode_slug(repo)))
```

Comment:
Is there a specific reason to use `=` for assignment here?

Reply:
ah no, just habit. I'll style it again